dask_expr/_collection.py

```diff
@@ -103,6 +103,62 @@ def _wrap_expr_op(self, other, op=None):
         )
 
 
+def _wrap_expr_method_operator(name, class_):
+    """
+    Add method operators to Series or DataFrame like DataFrame.add.
+    _wrap_expr_method_operator("add", DataFrame)
+    """
+    if class_ == DataFrame:
+
+        def method(self, other, axis="columns", level=None, fill_value=None):
+            if level is not None:
+                raise NotImplementedError("level must be None")
+            if not isinstance(other, DataFrame):
+                raise NotImplementedError("only other=DataFrame implemented")
+
+            axis = _validate_axis(axis)
+
+            if axis in (1, "columns"):
+                if isinstance(other, Series):
+                    msg = f"Unable to {name} dd.Series with axis=1"
+                    raise ValueError(msg)
+            return new_collection(
+                expr.MethodOperator(
+                    name=name,
+                    left=self.expr,
+                    right=other.expr,
+                    axis=axis,
+                    level=level,
+                    fill_value=fill_value,
+                )
+            )
+
+    elif class_ == Series:
+
+        def method(self, other, level=None, fill_value=None, axis=0):
```

Comment:
I think this should raise for axis=columns instead of the thing above?

Reply:
Maybe.. although I'm basing my opinion [from the current reference impl](https://github.com/dask/dask/blob/191d39177009d2cce25b818878118e35329b6db3/dask/dataframe/core.py#L4431), but that a series (self) can be used with both columns and index axis as it works with both so long as they're the same size. Does that seem correct?

Reply:
ok that's interesting, happy to leave as is then